src/offices/_single.html

```diff
@@ -26,12 +28,14 @@
         {% set show_subscription = intro.subscribe_form and intro.govdelivery_code %}
         <div class="content-l_col
                         {% if show_subscription -%}
```

Comment:
Yes, sounds good. Fixed in https://github.com/cfpb/cfgov-refresh/commit/eac9ef26c978cb35eb0dca58877b8668b02b9645